crates/auth/Cargo.toml

```diff
@@ -5,3 +5,22 @@ authors = ["Profian Inc"]
 edition = "2021"
 
 [dependencies]
+clap = { version = "3", features = ["derive"] }
+axum = { version = "0.4", features = ["headers"] }
+tokio = { version = "1.0", features = ["full"] }
+oauth2 = "4.1"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+reqwest = { version = "0.11", default-features = false, features = ["json"] }
+headers = "0.3"
+http = "0.2"
+rsa = "0.5"
+rand = "0.8"
+bincode = "1"
+base64 = "0.13"
+
+[dev-dependencies]
+hyper = "0.14"
+tower = "0.4"
+regex = "1"
+tokio-test = "0.4"
```

Comment:
Let's be consistent across crates and import like so https://github.com/profianinc/drawbridge/blob/88052ff31d2b2dfa6e164eb1f501ad6d96a0582d/crates/type/Cargo.toml#L21-L22